tests/cxx/default_tpl_arg.cc

```diff
@@ -43,17 +43,23 @@ struct Outer2 {
 // IWYU: IndirectTemplate needs a declaration
 Outer2<int, IndirectTemplate<int>> o2;
 
+void Fn() {
+  // Should not report IndirectClass because it is provided by template.
+  FnWithDefArgs1();
+  FnWithDefArgs2();
+}
+
 /**** IWYU_SUMMARY
 
 tests/cxx/default_tpl_arg.cc should add these lines:
 #include "tests/cxx/default_tpl_arg-i1.h"
 #include "tests/cxx/indirect.h"
 
 tests/cxx/default_tpl_arg.cc should remove these lines:
-- #include "tests/cxx/default_tpl_arg-d1.h"  // lines XX-XX
```

Comment:
Hmm, it's a little sad that this is no longer removed in favor of `-i1.h`. Could you instead add a new `-d2.h` header for the new test?